src/functions-and-arrays.js

```diff
@@ -52,16 +87,25 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
-
+function uniquifyArray(arr) {
+  if (arr.length === 0) {
+    return null;
+  } else {
+    return Array.from(new Set(arr));
```

Comment:
Nice! you could also the spread operator to turn the set into an array `[...new Set(arr)]`